parlai/mturk/core/mturk_manager.py

```diff
@@ -1071,14 +1073,15 @@ def start_new_run(self):
                 self.server_url,
                 self.task_group_id
             )
-        except Exception:
+        except Exception as e:
             self.topic_arn = None
             shared_utils.print_and_log(
                 logging.WARN,
                 'Botocore couldn\'t subscribe to HIT events, '
                 'perhaps you tried to register to localhost?',
                 should_print=True
             )
+            print(repr(e))
```

Comment:
is this redundant

Reply:
If referring to the `repr` call, `repr` actually gives more context on the error object than its tostring does. If referring to a print after the print and log, it's actually just in here for debug and I should remove before merging.